### ISSUE TITLE
Testing full storage usage of our servers.

### ISSUE THREAD

**Issue by naveen-fractlio** [fractalio/integralstor_gridcell#142]
We need to test the behavior of our storage system by dumping data which reaches full of the quota/space we set/have.

1. By this we can examine/analyze the Bottleneck and can set exact quota required for data-sets. 
2. By analyzing this we can give more space to customer as they needed. 
3. We can set standards of usage or allocation of space of our NAS. ex, for Distributed volumes, Replicated volumes.
4. Storage calculations for different types of volumes gives exact space utilization by which we can explain the customers easily.

For this test>>
1. We need to set testing environment with smaller size Data disk say 100GB/500Gb disks. 
2. Setup with Primary Secondary and a slave machine. Or Single Primary and secondary setup is enough.
3. Calculations made before and after dumping Data.
4. Analysis
5. Final Decision. 

Any Comments...???

**Comment by fractalram** [comment]
Good point Naveen.. In fact, one of the exercises that we need to do for both unicell and gridcell is to come up with a comprehensive list of test cases and keep adding to these test cases after every customer issue that we see. 

What you mention here will definitely have to be one of the primary testcases.
